### DISCUSSION TITLE
Node Map only includes Triliam Demo node

### DISCUSSION THREAD

**Discussion by mrickey** [TriliumNext/Notes#552] [Q&A / Support]
I downloaded Trilium today and am already planning to set up a server at home. I have been using Notion and Obsidian in the past.

My issue: When I select the tree view/node Map in the left tool bar, it seems to only show the "Trilium Demo" tree. I have added a couple of other trees at the root level. How do I get the other root-level nodes included?

Is there an easy way to import content from Obsidian and Notion?

Also, is there a discord server for TriliumNext?

Thanks in advance,
Michael

**Comment by meichthys** [comment]
We're glad you're giving TriliumNext a try!
I haven't seen the issue you're seeing with the Note Map. As you add notes/trees they should appear in the map. Perhaps the notes you've added are small so they aren't as noticeable in the map (the demo instance has lots of disjoint notes).

The easiest way to import from Obsidian or Notion would be to export the note content in Markdown (or html for better formatting), then zip the exports up into a zip file and import into TriliumNext (Right click any note and choose import). All the default import options should be good for most cases.

As for a chat server, we (in true OpenSource fashion) are hanging out over on Matrix: https://matrix.to/#/#triliumnext:matrix.org

**Reply by mrickey** [reply]
I am closing this ticket. I had too many problems getting some feature parity with Notion.

**Reply by eliandoran** [reply]
@mrickey , sorry to hear that. 

What features were you missing?

Either way, I would say comparing Trilium to either Notion or Obsidian is not the most fair comparison, in terms of how many people work on it and considering both Notion and Obsidian are proprietary software.

**Comment by mrickey** [comment]
My biggest single challenge is the lack of a mobile app. Also, no web
interface.

I understand that I can self-host this, but then I can only access it from
home. Not ideal.

Currently, I'm experimenting with LogSeq. I like the way it works, but I am
waiting for the web version as I don't like how the mobile app runs on my
Chromebook.

Their sync service works well for me...




*Michael Rickey* - AF6FB

949-298-0529 (*voice, text*)
***@***.***
http://mrickey.com

**Reply by eliandoran** [reply]
Just for reference, there's always the possibility of paid hosting such as [PikaPods](https://www.pikapods.com/apps) or trilium.cc (I believe that one doesn't run TriliumNext, but the original Trilium).

**Reply by meichthys** [reply]
There is a web interface when running the server: https://triliumnext.github.io/Docs/Wiki/server-installation.html
- The server can be accessed remotely by using a vpn or reverse proxy (requires some tech knowledge)
- You can also host the server on a cloud provider like pikapods: https://www.pikapods.com/apps (you can contact them via email [hello@pikapods.com](mailto:hello@pikapods.com) if the app page doesn't load the available apps - for some reason they don't load for me).

There is also an Android mobile app in development here: https://github.com/FliegendeWurst/TriliumDroid
